src/locale/locales/tr/messages.po

```diff
@@ -9332,7 +9332,7 @@ msgstr ""
 
 #: src/screens/Messages/ChatList.tsx:223
 msgid "You have no conversations yet. Start one!"
-msgstr ""
+msgstr "Henüz hiç görüşmeniz yok. Bir tane başlatın!"
```

Comment:
For the native turkish speakers:
`Start one!` için daha güzel bir önerisi olan var mı?

Reply:
"Hiçbir görüşme Yapmadınız. Şimdi Başlatınn!"

Reply:
"Henuz gorusmeniz bulunmamaktadir. Bir tane baslatin!" da uygun olabilir.